tests/test_other.py

```diff
@@ -9631,3 +9631,12 @@ def test_standalone_export_main(self):
     # point.
     # We should consider making this a warning since the `_main` export is redundant.
     self.run_process([EMCC, '-sEXPORTED_FUNCTIONS=[_main]', '-sSTANDALONE_WASM', '-c', path_from_root('tests', 'core', 'test_hello_world.c')])
+
+  def test_dash_o_output(self):
+    # Test that the file passed with -o is actually created.
+    self.run_process([EMCC, '-o', 'dash_o_output-1', path_from_root('tests', 'hello_world.c')])
+    self.assertTrue(os.path.exists('dash_o_output-1'))
```

Comment:
I don't think we need this first part.  We take this elsewhere in many places.

Reply:
Done.